cmd/nic-feature-discovery/main.go

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2023, NVIDIA CORPORATION & AFFILIATES
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/Mellanox/nic-feature-discovery/cmd/nic-feature-discovery/app"
+	"k8s.io/component-base/cli"
+	"k8s.io/klog/v2"
+)
+
+func main() {
+	klog.FlushAndExit(klog.ExitFlushTimeout, cli.Run(app.NewNICFeatureDiscoveryCommand()))
```

Comment:
`cli.Run` will flush logs, I think `klog.FlushAndExit` is not requried

Reply:
looking into Run implementation, yes i see it flushes them. 
will remove this one then.